src/openapi/peridio-admin-openapi.yaml

```diff
@@ -553,81 +459,76 @@ paths:
                 properties:
                   binary:
                     $ref: "#/components/schemas/binary"
-  /binaries/{binary_id}/purge:
-    post:
-      operationId: purge-a-binary
-      summary: Purge a binary
+  /binaries/{binary_prn}/parts/{index}:
+    put:
+      operationId: create-a-binary-part
+      summary: Create a Binary Part
       description: |
-        Deletes the binary content from Peridio while leaving the database record intact.
-
-        **DANGER:** This is a one-way, disruptive, and destructive action. See the
-        [binaries reference](/reference/binaries#purged-binaries) for information regarding the
-        impact of purging a binary.
+        Creates a binary part, check the documentation at TODO
```

Comment:
Do we have actual documentation to link to? 

What do you think we need to write here as a description?

Reply:
I think we can leave the description off this for now.